Entities/ACrab.h

```diff
@@ -515,6 +515,18 @@ int FirearmActivationDelay() const;
 	void SetLimbPathPushForce(float force);
 
 
+	/// <summary>
+	/// Gets this ACrab's stride sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this ACrab's stride sound.</returns>
+	SoundContainer * GetStrideSound() const { return m_StrideSound; }
+
+	/// <summary>
+	/// Sets this ACrab's stride sound.
```

Comment:
As well as the getter changes, these setters all need an `Ownership IS transferred!` at the end.
i.e. `/// Sets this ACrab's stride sound. Ownership IS transferred!`